cli/src/request.ts

```diff
@@ -165,16 +165,11 @@ export function eventStream(
       res.on('data', (chunk: Buffer) => {
         const lines = chunk.toString().trim().split(/\n\n/)
         for (const line of lines) {
+          // only passes along lines that start with "data:"
+          // otherwise, it will just ignore the line
```

Comment:
this is because we started adding IDs for resumption, right?

Reply:
No, our endpoints still only send `data:`-prefixed SSE messages. At one point I tried having our SSE streams send keepalives, but that broke the CLI because we were throwing if we saw any non-data messages. This change doesn't have us supporting the full SSE spec or anything, but it does allow SSE endpoints to send other types of messages without breaking the CLI.